src/utils/sauce-storage.js

```diff
@@ -0,0 +1,64 @@
+import Promise from 'pinkie';

```

Comment:
move out this file from utils. Make it `src/sauce-storage.js`.
Utils are usually common functions, helpers.